docuware-connector/src/com/axonivy/connector/docuware/connector/DocuWareAuthFeature.java

```diff
@@ -85,30 +90,56 @@ public void filter(ClientRequestContext reqContext) throws IOException {
 					}
 				}
 
-				if(StringUtils.isNotBlank(target)) {
-					Response response = reqContext.getClient()
-							.target(target)
-							.path(ACCOUNT_LOGON_PATH)
-							.request(MediaType.APPLICATION_FORM_URLENCODED_TYPE)
-							.accept(MediaType.APPLICATION_JSON)
-							.post(Entity.form(form));
-
-					Ivy.log().debug("Response: {0}", response);
-					if(getCookies() == null) {
-						Ivy.log().error("Still missing DocuWare cookies for context {0}", reqContext);
-					}
+				Ivy.log().info("DocuWare logon via URL {0}", logonUrl);
+
+				Response response = logon(reqContext, form, logonUrl);
+
+				Ivy.log().info("DocuWare logon returned with status {0}", response.getStatusInfo());
+
+				if(getCookies() == null) {
+					Ivy.log().error("DocuWare logon, still missing DocuWare cookies for context {0} after logon.", reqContext);
 				}
 			}
+
+			requestHook(reqContext);
 		}
 	}
 
+	protected Response logon(ClientRequestContext reqContext, Form form, String target) {
```

Comment:
cool: I think this is a good evidence of an initial test-problem, that actually lead to better productive code :+1: